staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -150,6 +151,8 @@ func getPublicIPDomainNameLabel(service *v1.Service) (string, bool) {
 
 // EnsureLoadBalancer creates a new load balancer 'name', or updates the existing one. Returns the status of the balancer
 func (az *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, service *v1.Service, nodes []*v1.Node) (*v1.LoadBalancerStatus, error) {
+	mc := metrics.NewMetricContext("operation", "ensure_loadbalancer", az.ResourceGroup, az.SubscriptionID, "")
```

Comment:
let's follow the patterns for other metrics, e.g. change this to `metrics.NewMetricContext("services", "create_or_update", ...)`. And we need to add service name in the metrics